src/RedisSessionStateProvider/KeyGenerator.cs

```diff
@@ -4,6 +4,7 @@
 //
 
 using System;
+using System.Web.SessionState;
```

Comment:
Is this new `using` statement necessary?

Reply:
removed